[0m[[0m[0mdebug[0m] [0m[0m> Exec(test, Some(68edab41-a194-4c2e-92fb-e293424cabcf), Some(CommandSource(console0)))[0m
[0m[[0m[0mdebug[0m] [0m[0mEvaluating tasks: Test / test[0m
[0m[[0m[0mdebug[0m] [0m[0mRunning task... Cancel: Signal, check cycles: false, forcegc: true[0m
[0m[[0m[0minfo[0m] [0m[0mcompiling 1 Scala source to C:\Users\SM\2021_Trading_System\target\scala-2.12\classes ...[0m
[0m[[0m[33mwarn[0m] [0m[0mC:\Users\SM\2021_Trading_System\src\main\scala\aia\integration\OrderServiceApi.scala:63:22: fruitless type test: a value of type scala.util.Try[Any] cannot also be a aia.integration.TrackingOrder[0m
[0m[[0m[33mwarn[0m] [0m[0m        case result: TrackingOrder =>[0m
[0m[[0m[33mwarn[0m] [0m[0m                     ^[0m
[0m[[0m[33mwarn[0m] [0m[0mC:\Users\SM\2021_Trading_System\src\main\scala\aia\integration\OrderServiceApi.scala:66:22: fruitless type test: a value of type scala.util.Try[Any] cannot also be a aia.integration.NoSuchOrder[0m
[0m[[0m[33mwarn[0m] [0m[0m        case result: NoSuchOrder =>[0m
[0m[[0m[33mwarn[0m] [0m[0m                     ^[0m
[0m[[0m[33mwarn[0m] [0m[0mC:\Users\SM\2021_Trading_System\src\main\scala\aia\integration\OrderServiceApi.scala:78:24: fruitless type test: a value of type scala.util.Try[Any] cannot also be a aia.integration.TrackingOrder[0m
[0m[[0m[33mwarn[0m] [0m[0m          case result: TrackingOrder =>[0m
[0m[[0m[33mwarn[0m] [0m[0m                       ^[0m
[0m[[0m[33mwarn[0m] [0m[0mC:\Users\SM\2021_Trading_System\src\main\scala\aia\integration\OrderServiceApi.scala:60:43: match may not be exhaustive.[0m
[0m[[0m[33mwarn[0m] [0m[0mIt would fail on the following inputs: Failure(_), Success(_)[0m
[0m[[0m[33mwarn[0m] [0m[0m      onComplete(processOrders.ask(item)) {   // onSuccessÎäî Ìì®Ï≤ò Í≤∞Í≥ºÎ•º ÎÇ¥Î∂Ä Î£®Ìä∏Ïóê Ï†ÑÎã¨Ìï®. Í∑∏Î¶¨Í≥† OrderIdÎ•º processOrders Ïï°ÌÑ∞ÏóêÍ≤å Î≥¥ÎÉÑ[0m
[0m[[0m[33mwarn[0m] [0m[0m                                          ^[0m
[0m[[0m[33mwarn[0m] [0m[0mC:\Users\SM\2021_Trading_System\src\main\scala\aia\integration\OrderServiceApi.scala:37:20: The outer reference in this type test cannot be checked at run time.[0m
[0m[[0m[33mwarn[0m] [0m[0m  final case class Item(name: String, id: Long)[0m
[0m[[0m[33mwarn[0m] [0m[0m                   ^[0m
[0m[[0m[33mwarn[0m] [0m[0mC:\Users\SM\2021_Trading_System\src\main\scala\aia\integration\OrderServiceApi.scala:38:20: The outer reference in this type test cannot be checked at run time.[0m
[0m[[0m[33mwarn[0m] [0m[0m  final case class Order(items: List[Item])[0m
[0m[[0m[33mwarn[0m] [0m[0m                   ^[0m
[0m[[0m[33mwarn[0m] [0m[0m6 warnings found[0m
[0m[[0m[0minfo[0m] [0m[0mshutting down sbt server[0m
